pkg/pac/pac.go

```diff
@@ -607,7 +608,18 @@ func GetPipelineNameFromMR() (pipelineName string) {
 	return pipelineName
 }
 
-func deleteGitlabProject(projectID int) error {
+func AssertPACInfoInstall() {
+	pacInfo, err := opc.GetOpcPacInfoInstall()
+	if err != nil {
+		testsuit.T.Fail(fmt.Errorf("failed to get pac info: %v", err))
+	}
+	if pacInfo.PipelinesAsCode.InstallVersion != os.Getenv("PAC_VERSION") &&
```

Comment:
I guess that you need to do substring check... variable contains something like `0.33` while cluster says `v0.33.0`